Explorer/Assets/Scripts/CrdtEcsBridge/RestrictedActions/RestrictedActionsAPIImplementation.cs

```diff
@@ -110,6 +112,25 @@ public bool TriggerSceneEmote(string src, bool loop)
             return true;
         }
 
+        public bool OpenNftDialog(string urn)
+        {
+            if (!sceneStateProvider.IsCurrent)
+            {
+                ReportHub.LogError(ReportCategory.RESTRICTED_ACTIONS, "OpenNftDialog: Player is not inside of scene");
```

Comment:
I see this pattern with the error everywhere: but are sure it's an error per se? Not a warning or info? Is it somehow restricted to the current scene on the SDK side? If not I think it's not an error

Reply:
In the old renderer we managed it as an error: https://github.com/decentraland/unity-renderer/blob/9e1a5dab078dd2ba29179c74ca6643bd5ac2eca3/browser-interface/packages/shared/apis/host/RestrictedActions.ts#L169-L172

In my opinion it should be an error since no action should be executed if we are outside the scene that executes it.

Anyway let me know wdyt and, if you think it should be a warning, we can change it without problem 👍